### PR TITLE
Wait for experiments library to complete loading

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

**Review by rchiodo — APPROVED** [review]

File: src/client/common/experiments/service.ts [hunk 1]
```diff
@@ -113,13 +118,15 @@ export class ExperimentService implements IExperimentService {
         // so we need to perform these checks and send the corresponding telemetry manually.
         switch (this.getOptInOptOutStatus(experiment)) {
             case 'optOut': {
+                await this.experimentationService.isCachedFlightEnabled(experiment);
```
Comment (DonJayamanne) [inline src/client/common/experiments/service.ts:121]: Regardless of whether user has opted in/out manually, we need to check with the experiment service.
This way if a user falls into an experiment, the telemetry will line up with experiments on server.

Comment (DonJayamanne) [reply src/client/common/experiments/service.ts:121]: @jakebailey @kimadeline /cc

Comment (DonJayamanne) [reply src/client/common/experiments/service.ts:121]: @rchiodo Please re-review.

File: src/client/common/experiments/service.ts [hunk 2]
```diff
@@ -120,6 +125,7 @@ export class ExperimentService implements IExperimentService {
                 return false;
             }
             case 'optIn': {
+                await this.experimentationService.isCachedFlightEnabled(experiment);
```
Comment (DonJayamanne) [inline src/client/common/experiments/service.ts:128]: Regardless of whether user has opted in manually, we need to check with the experiment service.
This way if a user falls into an experiment, the telemetry will line up with experiments on server.

**Review by kimadeline — APPROVED** [review]

**Review by DavidKutu — APPROVED** [review]

**Review by joyceerhl — APPROVED** [review]

**Pull request merged**